php/containers-schema.json

```diff
@@ -15,7 +15,7 @@
 					"image": {
 						"type": "string",
 						"minLength": 1,
-						"pattern": "^[a-z0-9/-]+$"
+						"pattern": "^(ghcr[.]io[/])?[a-z0-9-]+[/][a-z0-9-]+$"
```

Comment:
The reason that we dont allow ghcr.io is that we could not make the container update check work with ghcr.io https://github.com/nextcloud/all-in-one/blob/main/php/src/Docker/DockerHubManager.php

Reply:
If someone finds a solution then we can allow it :)